dkg-runtime-primitives/src/offchain/storage_keys.rs

```diff
@@ -18,3 +18,7 @@ pub const OFFCHAIN_SIGNED_PROPOSALS: &[u8] = b"dkg-proposal-handler::signed_prop
 
 // Key for offchain storage of aggregated derived public keys
 pub const AGGREGATED_MISBEHAVIOUR_REPORTS: &[u8] = b"dkg-metadata::misbehaviour";
+
+// Key for offchain storage of aggregated derived public keys
```

Comment:
This comment is not correct.
```suggestion
// Key for offchain storage lock while submitting signed proposals
```